src/PeraWalletConnect.ts

```diff
@@ -115,16 +117,11 @@ class PeraWalletConnect {
 
         document.getElementById("pera-wallet-iframe")?.remove();
       } else if (event.data.message.type === "CREATE_PASSCODE_EMBEDDED") {
-        const peraWalletConnectModal = document.getElementsByClassName(
-          "pera-wallet-connect-modal"
-        )[0];
+        const peraWalletConnectModal =
+          document.getElementsByClassName("pera-wallet-modal")[0];
```

Comment:
Let's please convert all of the IDs and classNames to constant so that we can make sure we're not messing up by changing a small className